teddy.js

```diff
@@ -382,11 +368,44 @@
 
       // compile template if necessary
       if (!teddy.templates[template] || teddy.params.compileAtEveryRender) {
-        renderedTemplate = teddy.compile(template)
+        renderedTemplate = teddy.compile(template, dontParse)
       }
 
       renderedTemplate = teddy.templates[template] || renderedTemplate
 
+      // check if noparse or noteddy tag exist
+      noparse = renderedTemplate.match(/noparse/g)
+      noteddy = renderedTemplate.match(/\snoteddy/g)
```

Comment:
these two regexes don't match. they either both need `\s` or neither do. (my guess is both do?)